### DISCUSSION TITLE
How can my form's state react to changes in other parts of the app?

### DISCUSSION THREAD

**Discussion by croraf** [react-hook-form/react-hook-form#9574] [Q&A] [answered]
Some part of my app that is not in the form changes and that should affect the state of the form.

How to handle that with react-hook-form?

**Comment by bluebill1049** [comment, marked as answer]
you can use setValue: https://react-hook-form.com/api/useform/setvalue method to update form values, but you will have to hoist useForm hook at the parent level to hoist the form and other part of app
